Kconfig

```diff
@@ -1473,6 +1473,10 @@ menu "LVGL configuration"
 		config LV_USE_EVDEV
 			bool "Use evdev input driver"
 			default n
+
+        config LV_USE_WINDOWS
```

Comment:
Use Tab for indentation in Kconfig,

Reply:
OK, I will update it. Please wait.

Kenji Mouri